yt_dlp/extractor/roosterteeth.py

```diff
@@ -133,10 +151,9 @@ def _real_extract(self, url):
 
         try:
             video_data = self._download_json(
-                api_episode_url + '/videos', display_id,
-                'Downloading video JSON metadata')['data'][0]
-            m3u8_url = video_data['attributes']['url']
-            # XXX: additional URL at video_data['links']['download']
+                api_episode_url + '/videos', display_id, 'Downloading video JSON metadata',
+                headers={'Client-Type': 'web'})['data'][0]
+            m3u8_url = traverse_obj(video_data, ('links', 'download')) or traverse_obj(video_data, ('attributes', 'url'))
```

Comment:
`video_data['links']['download']` URLs were giving 403 errors
```suggestion
                headers={'Client-Type': 'web'})['data'][0]
            m3u8_url = video_data['attributes']['url']
```